artemis-server/src/test/java/org/apache/activemq/artemis/core/persistence/impl/journal/OperationContextUnitTest.java

```diff
@@ -422,4 +422,43 @@ public void onError(final int errorCode, final String errorMessage) {
       Assert.assertEquals(0, operations.get());
    }
 
+   @Test
+   public void testContextWithoutDebugTrackers() {
+      ExecutorService executor = Executors.newSingleThreadExecutor(ActiveMQThreadFactory.defaultThreadFactory(getClass().getName()));
```

Comment:
You are not shutting down this executor. it's leaking a thread.

Reply:
add a runAfter(executor::shutdownNow);

Reply:
done